hpycc/utils/parsers.py

```diff
@@ -141,10 +140,7 @@ def parse_wuid_from_xml(result):
     return wuid.group(0)
 
 
-Schema = namedtuple("Schema", ["name", "is_set", "type"])
-
-
-def parse_schema_from_xml(xml):
+def parse_schema_from_xml(xml, dtype):
```

Comment:
is dtype just a way to override this?

Reply:
Yes, but on a per-column basis. I want to tell it column 20 is a string, I don't want to define the other 19 just so I can do this.

Reply:
Yes but on a per column basis. I don't want to enter all 20 columns just so I can change the type of one of them.